debian/changelog

```diff
@@ -0,0 +1,303 @@
+appcenter (3.2.4pop0) focal; urgency=medium
```

Comment:
This changelog has the incorrect package name, which prevents the package from building.